src/ansys/fluent/core/fluent_connection.py

```diff
@@ -315,7 +314,7 @@ def get_cortex_connection_properties(self):
             cortex_pid = cortex_info.process_id
             cortex_pwd = cortex_info.working_directory
             logger.debug("Cortex connection properties successfully obtained.")
-        except _InactiveRpcError:
+        except RuntimeError:
```

Comment:
Can we have a comment explaining why `RuntimeError` is the most appropriate choice here? Thanks.